wsclient/src/main/java/org/hpccsystems/ws/client/antlr/Ecl.g4

```diff
@@ -0,0 +1,53 @@
+grammar Ecl;		
```

Comment:
I chose to make it generic (as well as the object that it creates, EclInfo) so that if additional ecl was to be parsed in the future other than recordsets, the grammar name would remain the same. I can change it if you want.